include/is_sorted.hpp

```diff
@@ -22,7 +22,38 @@ namespace bit {
 template <class ForwardIt>
 constexpr bool is_sorted(bit_iterator<ForwardIt> first, 
     bit_iterator<ForwardIt> last) {
-    (first, last); 
+
+    using word_type = typename bit_iterator<ForwardIt>::word_type;
+    std::size_t num_digits = binary_digits<word_type>::value;
+    
+    bit_iterator<ForwardIt> cursor = first;
+    bool has_seen_1 = false;
+
+    while (cursor != last) {
```

Comment:
Same comment as before. While here we have Fwd iterators, it is still faster to work with word iterators. In general, I think `get_word` should only be used when we containers of different sizes. Otherwise, we can avoid the overhead by calling `_shld` or, even better, not do that at all. In this case and in `find`, we can can treat the first and last word as edge cases, and the words inbetween we can then just deal with directly, instead of having to shift for every word.